app/lib/features/events/widgets/skeletons/event_item_skeleton_widget.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:skeletonizer/skeletonizer.dart';
+
+class EventItemSkeleton extends StatefulWidget {
+  const EventItemSkeleton({super.key});
+
+  @override
+  State<EventItemSkeleton> createState() => _EventListSkeletonState();
+}
+
+class _EventListSkeletonState extends State<EventItemSkeleton> {
+  @override
+  Widget build(BuildContext context) {
+    return Skeletonizer(child: _buildEventItemSkeletonUI());
+  }
+
+  Widget _buildEventItemSkeletonUI() {
```

Comment:
:clap: again.

why is this a stateful widget though?

Why not just?

```suggestion
class EventItemSkeleton extends StatelessWidget {
  const EventItemSkeleton({super.key});
  
  @override
  Widget build(BuildContext context) {
```

Reply:
Opps! My Bad.
Yes it should be `StatelessWidget`. Let me change it.